idseq_dag/steps/generate_loc_db.py

```diff
@@ -19,10 +18,10 @@ def run(self):
         db_file = self.input_files_local[0][0]
         loc_db_file = self.output_files_local()[0]
         info_db_file = self.output_files_local()[1]
-        self.generate_loc_db(db_file, loc_db_file, info_db_file)
+        command.run_in_subprocess(PipelineStepGenerateLocDB.generate_loc_db)(db_file, loc_db_file, info_db_file)
 
-    @run_in_subprocess
```

Comment:
Why do you need to remove the decorator pattern?

Reply:
Because `get_context("spawn")` makes the subprocess run in a completely different memory space, and `pickle` complains about decorated function is not the same (like `Can't pickle <function xxxx at 0x123456>: it's not the same object`.
 I haven't dig too deep to check if there was a workaround for that though (maybe wrapping the wrapper would make `pickle` happy).
Running things in subprocesses using `spawn` is tricky because you cannot get access to any globals, closures and other shared objects (including for example decorated functions and example `multiprocessing.Value`. The last one it causes segmentation fault -11 when passed as a parameter to the subprocess).
Multiprocessing in Python3 seems to have a lot of bugs (take a look on [python3 changelog](https://docs.python.org/3/whatsnew/changelog.html) and search for `multiprocessing` to see how many entries do they have).
@boris-dimitrov suggested we should first try upgrading to Python3.7.X to see if it fix this underlying issue before merging this PR. I also agree with that approach.